### PR TITLE
Space-Travelers-Hub: Final PR

### PR DESCRIPTION
**Pull request by Kumilachew-g**
**In this project, we have done the following tasks.**

- Installed linter dependencies.
- Created the app and install basic dependencies.
- Downloaded the [free image](https://www.flaticon.com/free-icon/planet_3212567?term=space&page=1&position=19&page=1&position=19&related_id=3212567&origin=style) for the app logo.
- Created empty My profile.
- Created website header with empty nav.
- Created routes and view components (Rockets, Missions, My Profile]). Use **NavLink**  for the page navigation links and style active class to indicate which section/page user is currently on (underline active navigation link)
- Fetched data and updated Redux store.
- Once the data are fetched, dispatched an action to store the selected data in the Redux store:
- Used useSelector() Redux Hook to select the state slices and render lists of rockets and missions in corresponding routes.
- Created basic structure for Missions
- When a user clicked the "Reserve rocket" button, action needed to be dispatched to update the store.
- Created a reducer and action dispatcher for the "Join Mission" button.
- Created basic structure for Rockets.
- Rockets that have already been reserved showed a "Reserved" badge and a "Cancel reservation".
- Missions that the user has joined already showed a badge "Active Member" instead of the default "NOT A MEMBER" and a button "Leave Mission" instead of the "Join Mission" button (as per design).
- Added display rockets functions.
- Added display missions functions.
- Implemented mission leaving - Actions.
- Implemented rocket booking - Actions.
The Space Travelers' Hub consists of Rockets, Missions, and the My Profile section.

[**Rockets**](https://space-travelerskp-hub.netlify.app/)

 The Rockets section displays a list of all available SpaceX rockets. Users can book each rocket by clicking the reservation button or cancel the previously made booking.

image

[**Missions**](https://space-travelerskp-hub.netlify.app/missions)

The Missions section displays a list of current missions along with their brief description and participation status. There is also a button next to each mission that allows users to join the selected mission or leave the mission the user joined earlier.

image

[**My Profile**](https://space-travelerskp-hub.netlify.app/profile)

The My Profile section displays all reserved rockets and space missions.

image

### CODE REVIEW

File: src/components/navigation/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+const NavBar = () => (
+  <>
+    <Navbar className="border-bottom">
+      <Container className="nav-wrapper">
+        <Navbar.Brand href="/">
+          <img src={logo} alt="Logo" className="logo" />
+          <h1>Space Travelers Hub</h1>
+        </Navbar.Brand>
+        <Nav className="nav-links p-3">
```
Comment (tufoinnkuo10) [inline src/components/navigation/NavBar.js:15]: - [ ] Great job so far. design looks great. please ensure that the active page is  indicated with an underline
Screenshot (484)_LI

File: src/components/rockets/ProfileRocket.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const ProfileRocket = () => {
+  let status = null;
+  const rocket = useSelector((state) => state.Rockets);
+  const filteredRocket = rocket.filter((rocket) => rocket.reserved === true);
+  if (!filteredRocket.length) { status = 'No Rockets reserved'; }
+  return (
+    <>
+      <div className="profile-rocket-container">
+        <h2>My Rockets</h2>
+        <ul className="profile-rocket-list-conatiner">
```
Comment (tufoinnkuo10) [inline src/components/rockets/ProfileRocket.js:15]: - [ ] everything looks fine. Please add a little margin bottom/top to the images, so they can easily be differentiated.
Screenshot (485)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @TEAM

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Design well implemented
✅ Test ok

## Required Changes ♻️
Check the comments under the review

tufoin


Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Kumilachew-g** [comment]
**I have done the comments based on the reviewer**

Rocket UI

image

Mission UI
image

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Kumilachew-g , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

### Highlights:
 ✔️ Project is setup correctly
 ✔️ All linters are passing
 ✔️ Correct gitflow
 ✔️ Professional documentation

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

image

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@shoirata] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**